libnavui-voice/src/test/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayerTest.kt

```diff
@@ -63,13 +65,35 @@ class VoiceInstructionsTextPlayerTest {
         assertEquals(null, textPlayer.currentPlay)
     }
 
+    @Test
+    fun `language is not supported and checkIsLanguageAvailable set to false current play not null`() {
```

Comment:
It says `current play not null` but we're asserting https://github.com/mapbox/mapbox-navigation-android/blob/fdfd1ab7512f9e38577c55310319553249b0a945/libnavui-voice/src/test/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayerTest.kt#L87 What am I missing?

Reply:
Will fix it